app/graphql/types/custom/plan_detail_type.rb

```diff
@@ -3,12 +3,13 @@ module Types
   module Custom
     class PlanDetailType < Types::BaseObject
       description "Represents a single detail in a Budget Plan"
+      field :budget_plan_id, ID, null: true, description: "FK from Budget_Plans"
       field :id, ID, null: true, description: "Auto Generated ID"
-      field :category_id, ID, null: true, description: "FK from Categories"
       field :amount, Float, null: true, description: "The amount this user budgeted on this Category"
-      field :budget_plan_id, ID, null: true, description: "FK from Budget_Plans"
-      field :category, String, null: true, description: "FK from Budget_Plans"
-      field :short_desc, String, null: true, description: "FK from Budget_Plans"
+      field :category, Types::Custom::CategoryType, null: true, description: "Category details"
```

Comment:
I forgot to remove these comments I tried to push a change on top of this that didnt work  :(

Reply:
what error are u getting?

Reply:
it wasnt an error just left old code commented out , it should have been cleaned up